core/storage/storage_models_test.py

```diff
@@ -62,9 +64,8 @@ def test_all_model_module_names_unique(self):
             len(set(names_of_ndb_model_subclasses)),
             len(names_of_ndb_model_subclasses))
 
-    @unittest.skip('get_deletion_policy is not yet implemented on all models')
     def test_all_models_have_get_deletion_policy(self):
-        model_subclasses = self._get_model_classes()
+        model_subclasses = self._get_model_classes(skip_base_models=True)
```

Comment:
I suggest removing the changes which add skip_base_models -- I don't think this warrants adding lots of args to functions. Couldn't you just exclude base_model here using an if statement in the body of the for loop, perhaps?

Reply:
I would have to skip all the base classes here by explicitly listing their names. This arg will also be used for testing if all models have `has_reference_to_user_id`.

Reply:
Then create a constant whitelist of classes to skip, and use that instead to filter (so, explicitly listing their names is indeed the way to go).